### PR TITLE
Migrate org types

### PR DESCRIPTION
**Pull request by CrystalPea**
### In this PR:
- Migrate all orgs and services onto new organisation types
- Remove all mentions of generic `'nhs'` organisation type.

Part of this story: 
https://www.pivotaltracker.com/story/show/166666407

### CODE REVIEW

**Comment by CrystalPea** [comment]
@servingUpAces I will check

File: migrations/versions/0300_migrate_org_types.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+revision = '0300_migrate_org_types'
+down_revision = '0299_org_types_table'
+
+environment = os.environ['NOTIFY_ENVIRONMENT']
+
+
+def upgrade():
+    if environment != "production":
```
Comment (servingUpAces) [inline migrations/versions/0300_migrate_org_types.py]: You need to include 'live' as well

Comment (CrystalPea) [reply migrations/versions/0300_migrate_org_types.py]: Good call, I've done that now :)

**Comment by CrystalPea** [comment]
@servingUpAces 'nhs' was not added to organisation_types table, it was only in a list in models that the valdiation was done against. This PR removes it completely form API code.

**Pull request merged**